package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "k6-studio",
   "productName": "k6 Studio",
-  "version": "0.12.0",
+  "version": "0.13.0",
```

Comment:
This doesn't seem right 👀

Reply:
this is just for testing and will be reverted before the PR gets merged 🙇